harness/determined/common/api/analytics.py

```diff
@@ -1,21 +1,46 @@
+import logging
 import os
+import sys
+from typing import Any, Dict, Type
 
 import analytics
 
+enabled = os.environ.get("DET_SEGMENT_ENABLED") == "true"
```

Comment:
Can I request that we start using environment variables to represent bools differently?

With the vast majority of projects I use, they say something like, 'set `ENABLE_XYZ` to any non-empy string to enable XYZ".  We instead inspect the value of the environment variable, even when the variable is just a flag value.

Can we switch to treating new environment variables as true when they are any non-empty string, when they are bools like this?

Reply:
Yes. And I think for such a switch we should first write it down in our coding style guideline and then use a separate PR to make the system consistent in using that style.